reactioncommerce/reaction
# refactor: use npm package for products plugin

Resolves https://github.com/reactioncommerce/reaction/issues/6177

This PR removes the internal `products` plugin, and installs the new `api-plugin-products` from npm as a replacement.

The code in that npm plugin is the same, so the expected outcome of this PR is seeing nothing different than how the app currently runs.

## Testing
- Start reaction, see that `products` is the correct version in the `systemInformation` query
- Create or update a product

Signed-off-by: Maria Paktiti <maria.paktiti@gmail.com>
